db/multi-table.sql

```diff
@@ -62,7 +62,7 @@ CREATE TABLE dragonfruit (
     uptime_ms BIGINT,
     batt_mv INTEGER,
     panel_mv INTEGER,
-    apogee_sp215 INTEGER,
+    apogee_sp215 FLOAT,
```

Comment:
I guess I didn't notice this before -- maybe you should also change `apogee_sp215` in the dragonfruit board to match the other ones as well.... was there a reason for the difference in naming scheme? I can't remember haha.